packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -43,4 +52,123 @@ describe('StatelessVerkleStateManager: Kaustinen Verkle Block', () => {
       'should have correct codeHash'
     )
   })
+
+  it('put/delete/modify account', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = new Address(randomBytes(20))
+
+    try {
+      await stateManager.getAccount(address)
+      assert.fail('should throw on getting account that is not found in witness')
+    } catch (e) {
+      assert.equal(
+        e.message.slice(0, 25),
+        'Missing execution witness',
+        'should throw on getting account that does not exist in cache and witness'
+      )
+    }
+
+    const account = Account.fromAccountData({
+      nonce: BigInt(2),
+    })
+    await stateManager.putAccount(address, account)
+
+    let check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.modifyAccountFields(address, {
+      nonce: BigInt(2),
+    })
+    check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.deleteAccount(address)
+
+    check = await stateManager.getAccount(address)
+    assert.isUndefined(check, 'should return undefined for deleted account')
+  })
+
+  it('getTreeKeyFor* functions', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = Address.fromString('0x9791ded6e5d3d5dafca71bb7bb2a14187d17e32e')
+    const stem = getStem(address, 0)
+
+    const balanceKey = stateManager.getTreeKeyForBalance(stem)
+    const nonceKey = stateManager.getTreeKeyForNonce(stem)
+    const codeHashKey = stateManager.getTreeKeyForCodeHash(stem)
+
+    const balanceRaw = stateManager['_state'][bytesToHex(balanceKey)]
+    const nonceRaw = stateManager['_state'][bytesToHex(nonceKey)]
+    const codeHash = stateManager['_state'][bytesToHex(codeHashKey)]
+
+    const account = await stateManager.getAccount(address)
+
+    assert.equal(
+      account.balance,
+      bytesToBigInt(hexToBytes(balanceRaw), true),
+      'should have correct balance'
+    )
+    assert.equal(
+      account.nonce,
+      bytesToBigInt(hexToBytes(nonceRaw), true),
+      'should have correct nonce'
+    )
+    assert.equal(bytesToHex(account.codeHash), codeHash, 'should have correct codeHash')
+  })
+
+  it(`copy()`, async () => {
+    const sm = new StatelessVerkleStateManager({
```

Comment:
Nit, but imo let's name this consistently throughout tests (so stateManager instead of sm) 
```suggestion
    const stateManager = new StatelessVerkleStateManager({
```

Reply:
Fixed in a separate commit.